yt_dlp/extractor/rtvslo.py

```diff
@@ -0,0 +1,150 @@
+                'description': 'md5:76a18692757aeb8f0f51221106277dd2',
+                'timestamp': 1643137046,
+                'title': 'Dnevnik',
+                'series_id': '92',
+                'release_date': '20220125',
+                'duration': 1789,
+            },
+        }, {
+            'url': 'https://365.rtvslo.si/arhiv/utrip/174843754',
+            'info_dict': {
+                'id': '174843754',
+                'ext': 'mp4',
+                'series_id': '94',
+                'release_date': '20220129',
+                'timestamp': 1643484455,
+                'title': 'Utrip',
+                'duration': 813,
+                'thumbnail': 'https://img.rtvcdn.si/_up/ava/ava_misc/show_logos/94/utrip_1_wide2.jpg',
+                'description': 'md5:77f2892630c7b17bb7a5bb84319020c9',
+                'release_timestamp': 1643485825,
+                'upload_date': '20220129',
+                'series': 'Utrip',
+            },
+        }, {
+            'url': 'https://365.rtvslo.si/arhiv/il-giornale-della-sera/174844609',
+            'info_dict': {
+                'id': '174844609',
+                'ext': 'mp3',
+                'series_id': '106615841',
+                'title': 'Il giornale della sera',
+                'duration': 1328,
+                'series': 'Il giornale della sera',
+                'timestamp': 1643743800,
+                'release_timestamp': 1643745424,
+                'thumbnail': 'https://img.rtvcdn.si/_up/ava/ava_misc/show_logos/il-giornale-della-sera_wide2.jpg',
+                'upload_date': '20220201',
+                'bitrate': 128000,
+                'release_date': '20220201',
+            },
+
+        }, {
+            'url': 'https://4d.rtvslo.si/arhiv/dnevnik/174842550',
+            'only_matching': True
+        }
+    ]
+
+    def _real_extract(self, url):
+        v_id = self._match_id(url)
+        meta = self._download_json(self._API_BASE.format('getRecordingDrm', v_id), v_id).get('response')
+        date = unified_timestamp(meta.get('broadcastDate') or meta.get('broadcastDates')[0])
+
+        thumbs = [{'id': k, 'url': v} for (k, v) in meta.get('images').items()]
+        SUB_LANGS_MAP = {'Slovenski': 'sl', }
+
+        subs = {}
+        for s in traverse_obj(meta, 'subs', 'subtitles', default={}):
+            if s.get('language') in SUB_LANGS_MAP.keys():
+                s['language'] = SUB_LANGS_MAP[s['language']]
+            if not subs.get(s.get('language'), False):
+                subs[s.get('language')] = []
+            subs[s.get('language')].append({'url': s.get('file'), 'format': s.get('format').lower()})
+
+        jwt = meta.get('jwt')
+        if not jwt:
+            raise ExtractorError('Site did not provide an authentication token, cannot proceed.')
+
+        media = self._download_json(self._API_BASE.format('getMedia', v_id) + f'&jwt={jwt}', v_id).get('response')
```

Comment:
```suggestion
        media = self._download_json(self._API_BASE.format('getMedia', v_id), v_id, query={jwt: jwt}).get('response')
```

Reply:
Similar to above, decide whether this should be fatal or not! Just using `.get` without other precautions don't accomplish anything